jobs/build/update-3.x-base-images/README.md

```diff
@@ -0,0 +1 @@
+Parameterized job for running update 3.x base images.
```

Comment:
We're going to need this for 4.x base images too, so it should be named generically.
I think at the moment that only means adding `ubi7` to the list (should be just like `rhel7`).